index.html

```diff
@@ -0,0 +1,153 @@
+                <h2 class="date">2023.04.03(MON) ~ 6(THUR)</h2>
+                <p class="venue">@ Nairobi National Museum Nairobi, Kenya.</p>
+            </div>
+        </div>
+
+        <div class="main-program">
+            <!-- dark background -->
+            <!-- <a href="https://www.freepik.com/free-photo/solid-concrete-wall-textured-backdrop_17839221.htm#query=dark%20background&position=17&from_view=search&track=sph">Image by rawpixel.com</a>  -->
+
+            <h3>Main Program</h3>
+            <div class="program-cards-layout">
+                <div class="program-card">
+                    <i class="fa-regular fa-handshake"></i>
+                    <h3 class="card-title">Meet the Artists</h3>
+                    <p class="card-desc">
+                        An opening night event where visitors can meet the artists, learn about their work and processes, and engage in Q&A sessions.
+                    </p>
+                </div>
+                <div class="program-card">
+                    <i class="fa-sharp fa-solid fa-chalkboard"></i>
+                    <h3 class="card-title">Workshop Series</h3>
+                    <p class="card-desc">
+                        A series of interactive workshops led by the artists, covering a variety of topics such as painting techniques, printmaking, and sculpture.
+                    </p>
+                </div>
+                <div class="program-card">
+                    <i class="fa-solid fa-square-poll-horizontal"></i>
+                    <h3 class="card-title">Artist Talks</h3>
+                    <p class="card-desc">
+                        An opportunity for artists to give presentations about their work and share their experiences and insights with visitors.
+                    </p>
+                </div>
+                <div class="program-card">
+                    <i class="fa-solid fa-eye"></i>
+                    <h3 class="card-title">Art Tour</h3>
+                    <p class="card-desc">
+                        A tour of the galleries, allowing visitors to get a closer look at the artwork and learn more about the artists and their work.
+                    </p>
+                </div>
+                <div class="program-card">
+                    <i class="fa-regular fa-circle-xmark"></i>
+                    <h3 class="card-title">Closing Celebration</h3>
+                    <p class="card-desc">
+                        A finale event with a closing reception and the announcement of awards for the most exceptional artwork.
+                    </p>
+                </div>
+            </div>
+
+            <a href="#" class="join">Join Global art Exhition 2023</a>
+            <a href="#" class="whole-program"> SEE  THE  WHOLE  PROGRAM</a>
+
+        </div>
+
+        <div class="speakers">
+            <h3 class="featured-speakers">Featured Speakers</h3>
+
+            <div class="speakers-cards-layout">
+
+                
+            </div>
```

Comment:
- [ ] Kindly make the speaker image bigger and add the other background(that is similar to a chess board) to match the Behance design.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/217524709-43ede6fd-5499-4b98-850a-565e66c78003.png) | ![image](https://user-images.githubusercontent.com/24703171/217524859-a832f98a-253d-4167-a457-38f21b3d1ae0.png) |

Reply:
oh yeah, I will adjust the image sizes immediately